star_name.rb

```diff
@@ -0,0 +1,38 @@
+require 'terminal-table'
+require 'colorize'
+
+
+class SolarSystem
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+  attr_reader :star_name, :planets
```

Comment:
By convention this should be above `initialize`.  Ruby doesn't care, but it's more of a style issue.